Support/grdb_config.c

```diff
@@ -1,5 +1,10 @@
 #include <GRDB/grdb_config.h>
-#include <GRDB/sqlite3.h>
+#if SQLITE_HAS_CODEC
+    #include <SQLCipher/sqlite3.h>
```

Comment:
I'd like to see a `#ifdef COCOAPODS` guard the inclusion of `<SQLCipher/sqlite3.h>`.